### ISSUE TITLE
Spiders have been breeding in tauka's mom's kitchen

### ISSUE THREAD

**Issue by LZakida** [addictgamer/Terrastation-13#34]
lol

But really now there is a random event that will attempt a spider outbreak in the incinerator room, which rock2 does not have. I actually noticed this bug long before with other types of pest outbreaks; none of them give an eighth of a fuck if that area actually exists or not, but I only just remembered this would happen as of late during a 1-3 player LAN stresstest. Is this something we want to try and address? 
(or do we just not really give a fuck)

**Comment by taukausanake** [comment]
Why is there not an incinerator room? That's one of the places where you get rid of dead bodies as well as an area that doesn't normally get viewed as much. Also a ghetto Toxins room

**Comment by LZakida** [comment]
Short answer: No room.

Long Answer: Since there is already a disposal area of sorts on the station and also since I _do not actually know_ how the incinerator area works ingame plus the fact that I didn't realize the area existed in the first place when I initially designed the rough area layout when I first began the project, I discovered _after I had already finished the station's structure_ that I did not have the area installed and no longer had a place to put it without ripping out a substantial amount of shit and/or reworking it. At that point, I decided to open an issue with it to hear others' thoughts on the matter.

You are now up to speed
